public/server/db.json

```diff
@@ -97,7 +97,7 @@
     },
```

Comment:
Oops. I think there is a trailing bracket here.
```suggestion
    
```

Reply:
Thank you for your encouragement! 

Looking at "Byob" that starts from line 94 and ends at line 97, it looks all good?

Reply:
> Thank you for your encouragement!
> 
> Looking at "Byob" that starts from line 94 and ends at line 97, it looks all good?

Ah yes. Looks like a miscalculation on my part. You have a good eye haha. Thanks again for the contribution! I'll go ahead and merge this.